patches/@react-native-community+netinfo+11.2.1+002+turbomodule.patch

```diff
@@ -2609,32 +2619,6 @@ index 6982220..b515270 100644
 +    readonly eventEmitter: NativeEventEmitter;
  };
  export default _default;
-diff --git a/node_modules/@react-native-community/netinfo/package.json b/node_modules/@react-native-community/netinfo/package.json
```

Comment:
Hey @OlimpiaZurek what is the reason for this change? `codegen` config is necessary for newArch builds

Reply:
@MrRefactor This change was applied automatically when patching the `netinfo` library using the `patch-package`. I checked and it looks like netinfo supports new arch, so it should work for newArch builds.

Reply:
based on this PR https://github.com/react-native-netinfo/react-native-netinfo/pull/655 that change is necessary, for me without that change, 0.74 builds are failing

Reply:
Got it. I will prepare a PR to reverse this change.